R/test.data.table.R

```diff
@@ -79,6 +79,9 @@ test.data.table = function(script="tests.Rraw", verbose=FALSE, pkg=".", silent=F
   # sample method changed in R 3.6 to remove bias; see #3431 for links and notes
   # This can be removed (and over 120 tests updated) if and when the oldest R version we test and support is moved to R 3.6
 
+  userNumericRounding = getNumericRounding()
+  setNumericRounding(0)  # Initialise to 0 in case the user has set it to a different value. Restore to user's value when finished.
```

Comment:
we should probably return the old rounding instead of `NULL` in `setNumericRounding()`, to match similar behavior by `setwd()`, `options()`, etc. Out of scope for this PR.

Reply:
#6108